presto-native-execution/presto_cpp/main/CMakeLists.txt

```diff
@@ -32,6 +31,15 @@ add_library(
   TaskManager.cpp
   TaskResource.cpp)
 
+add_library(presto_adapters INTERFACE)
```

Comment:
do we need the presto_adapters library?
velox_hive_connector library is sufficient?

Reply:
@majetideepak You are right, I have tested it in the local environment, and removing these also works